autorest/codegen/models/operation_group.py

```diff
@@ -97,6 +97,8 @@ def imports(self, async_mode: bool, has_schemas: bool) -> FileImport:
     @property
     def filename(self) -> str:
         if self.code_model.options["combine_operation_files"]:
+            if self.code_model.options["add_python_3_operation_files"]:
```

Comment:
@msyyc if `--add-python3-operation-files` is `True`, we will have 2 files, one called `_operations`, and the other called `_operations_py3`, so I don't think it's right to only return `_operations_py3` in the case of `add-python3-operation-files`. I think it's fine as is, if you really want to, you could do a name change, but I'm not sure it's worth it rn